revisions/J0/src/data/backgrounds/background_tile_commands.asm

```diff
@@ -1,7 +1,7 @@
 ; File generated automatically by `tools/generate_background_data.py`
 
 
-BackgroundTileCommands24 ; $4DD4
+BackgroundTileCommands24:: ; $4DD4
```

Comment:
oh, and this is an old style label which will be removed from rgbds in the future, gives a compile warning.